platforms/substrate/configuration/samples/network-example.yaml

```diff
@@ -48,6 +48,7 @@ network:
     command: "substrate"     # Please ensure the command corresponds to the node_image above    
     # NOTE for the below paths, the directories should exist
     bootnodes: "/home/CHANGE_ME/bevel/build/substrate_bootnodes" # Location where bootnodes information is read or saved if empty
+    bootnode_id: "/home/CHANGE_ME/bevel/build/bootnode_id" # location where bootnode's peer id is saved or read from
```

Comment:
Why is this needed separately? The substrate_bootnodes file will contain the full dns4 or ipv4 details